dashboards-observability/public/components/visualizations/charts/bar/bar.tsx

```diff
@@ -37,6 +37,14 @@ export const Bar = ({ visualizations, layout, config }: any) => {
   const isVertical = barOrientation === vis.orientation;
   let bars, valueSeries, valueForXSeries;
 
+  const [showInputBox, setShowInputBox] = useState<boolean>(false);
```

Comment:
all state variable can be combined in to one object and we can have one state variable.

Reply:
Done